cmd/bolt/main.go

```diff
@@ -653,7 +714,14 @@ func (cmd *PageCommand) PrintPage(w io.Writer, r io.ReaderAt, pageID int, pageSi
 // Usage returns the help message.
 func (cmd *PageCommand) Usage() string {
 	return strings.TrimLeft(`
-usage: bolt page -page PATH pageid [pageid...]
+usage: bolt page PATH pageid [pageid...]
```

Comment:
`DumpCommand`'s Usage looks similarly broken, might as well fix that too...

Reply:
Sure enough. Fixed.